tests/Tests/100__deploy/100__installation/109__operator.robot

```diff
@@ -24,6 +24,19 @@ Verify Odh-deployer Checks Cluster Platform Type
     ${odhdeployer_logs}=    Fetch Odh-deployer Pod Logs
     Should Contain    ${odhdeployer_logs}    ${odhdeployer_logs_content}
 
+Verify That The Operator Pod Does Not Get Stuck After Upgrade
+    [Documentation]    Verifies that the operator pod doesn't get stuck 
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-818
+    ${operator_pod_info}=    Fetch operator Pod Info
+    ${length}=    Get length    ${operator_pod_info}
+    IF    ${length} == 2
+            ${err_image_pull}=    Verify Operator Pods Has ErrImagePull After upgrade    ${operator_pod_info}
```

Comment:
if length is 2, then it means there are two instances of the operator pod running at the same time, right? If yes, then should line 35 be run for both the pods?

Reply:
The keyword `Verify Operator Pods Has ErrImagePull After upgrade `  contains `OpenShift Resource Field Value Should Be Equal As Strings` that run in both pods and look for the error in each pod.

Reply:
right, sorry stupid question, i missed something xD thx